ops/ansible/roles/bfd-server/tasks/install.yml

```diff
@@ -71,6 +71,16 @@
     mode: u=rw,g=r,o=r
   become: true
 
+- name: Copy New Relic Java Agent Config
+  copy:
+    src: 'newrelic.yml.j2'
+    dest: "{{ data_server_dir }}/newrelic/newrelic.yml"
+    remote_src: true
+    owner: "{{ data_server_user }}"
+    group: "{{ data_server_user }}"
+    mode: u=rw,g=r,o=r
```

Comment:
If this file will contain any api keys or what not, I would remove `o=r`.

Reply:
Right now intending it to not include the API key, but will take this out to be safe.